src/TransactionTestsFiller/ttGasLimit/TransactionWithHighGasFiller.json

```diff
@@ -1,26 +1,21 @@
 {
-    "TransactionWithHighGas" : {
-	"expect" : [
-            {
-                "network" : ["Frontier"],
-                "result" : "valid"
-            },
-	    {
-                "network" : ["Homestead", "EIP150", "EIP158", "Byzantium", ">=Constantinople"],
-                "result" : "invalid"
-            }
-        ],
+    "TransactionWithHighGas" :
+    {
+	    "expectException" : 
+        {
+            ">=Homestead" : "InvalidGasLimit4"
+        },
         "transaction" :
         {
             "data" : "",
-	    "gasLimit" : "0x7fffffffffffffff",
+	        "gasLimit" : "0x7fffffffffffffff",
```

Comment:
this test is identical to `TransactionWithHihghGasLimit63m1Filler.json `
better change to   x, x-1, x+1(if x not fails)  where x is gasLimit cap  (pow(2,63)-1)